tests/backup/backup_upgrade_test.go

```diff
@@ -368,6 +401,29 @@ var _ = Describe("{PXBackupEndToEndBackupAndRestoreWithUpgrade}", Label(TestCase
 				dash.VerifyFatal(err, nil, fmt.Sprintf("Verifying creation of backup location [%s] with UID [%s] using the bucket [%s]", backupLocationName, backupLocationUid, bucketName))
 			}
 		})
+
+		Step("Create cloud credentials and backup locations for partial backup validation", func() {
```

Comment:
Why we need separate backup location here?

Reply:
The plan was to not to touch or make use of any of the existing backup objects.